scripts/build.ts

```diff
@@ -19,7 +19,7 @@ await build({
   },
   entryPoints: srcFiles.map((file) => new URL(file, root)).map(fileURLToPath),
   format: 'esm',
-  minify: true,
+  // minify: true,
```

Comment:
I'm curious, what was the reason for disabling this? Was minified code problematic for debugging?

Reply:
This is a debug leftover. Though, the minification causes Vite to hiccup on sourcemaps for some reason, so all the lines are messed up. That's why I'm considering disabling this option until this bug is fixed. What do you think?

Reply:
> This is a debug leftover. Though, the minification causes Vite to hiccup on sourcemaps for some reason, so all the lines are messed up. That's why I'm considering disabling this option until this bug is fixed. What do you think?

Agree. Also bug reports with real thing names in the error messages and stack traces are much nicer than what we have now with mangled names.

Reply:
It's fine to address minification disabling in a separate PR though.

Reply:
Yep, will do a separate one

Reply:
Btw, mangled names are a bug I believe. I know why it happened and it would be better to fix it eventually.